src/pages/Login/Login.tsx

```diff
@@ -11,6 +12,7 @@ export const Login = () => {
     <Page classes="login">
       <div className="login-page__app-title">{title}</div>
       <div className="login-page__form-container">
+        <LanguageSwitcherContainer></LanguageSwitcherContainer>
```

Comment:
Please use a single self closed tag, if it doesn't contains children:
```<LanguageSwitcherContainer/>```

Reply:
Fixed!